### PR TITLE
feat: new gameplay instructions and prompts

### PR DESCRIPTION
**Pull request by ryokowo**

### CODE REVIEW

**Review by maelisrenau — APPROVED** [review]
ok great work !!

File: README.md
[the remark this answers is not in the document]
Comment (ryokowo) [reply README.md:1]: Yes, I believe the changes will create new dynamics to the game, making it more amusing.

**Comment by davimarim** [comment]
I changed the system of points. What do you think about it? @ryokowo @maelisrenau

File: README.md
[the remark this answers is not in the document]
Comment (davimarim) [reply README.md:1]: Yeah, it improved a lot!

**Pull request merged**